native/src/routes/jpal-tracking/containers/JpalTrackingContainer.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import type { TFunction } from 'react-i18next'
+import { withTranslation } from 'react-i18next'
+import withTheme from '../../../modules/theme/hocs/withTheme'
+import JpalTracking, { type PropsType as JpalTrackingPropsType } from '../components/JpalTracking'
+
+export default withTheme<$Diff<JpalTrackingPropsType, {| t: TFunction |}>>(
+  withTranslation<JpalTrackingPropsType>('settings')(JpalTracking)
+)
```

Comment:
I'm white sure you could use here hooks of react-i18n and our theme.
That would make the tracking screen a container, but I thinkg this is fine.

Reply:
changed it for translations, for theme we would need a ThemeProvider, we do not have yet, as Steffen poineted out.